latex/non-integral.tex

```diff
@@ -2,7 +2,7 @@ \section{Non-Integral Calculations}
 \label{sec:non-integr-calc}
 
 In the ledger there are several cases where non-integral calculations are
-required. This does concern the delegation transitions, not value transactions.
+required. This concerns the delegation transitions, but not the value transactions.
```

Comment:
not sure about this - and "value transactions" ?

Reply:
hmmm, this sentence is a bit odd (perhaps just outdated). There are no "delegations transactions" or "value transactions" per say. It just that the "delegation components/parts" of a transaction are the culprits for the non-integral calculations, and not the UTxO components.